### PR TITLE
sql midifications due to name change

### PR DESCRIPTION
**Pull request by ev1lm0nk3y**
SQL instance names cannot exceed 32 characters, so production be came prod.

### CODE REVIEW

File: deployment/gcp/sql.tf [hunk 1]
```diff
@@ -20,7 +20,7 @@ resource "google_sql_database_instance" "ticket_booth" {
       value = "on"
     }
 
-    deletion_protection_enabled = true
+    # deletion_protection_enabled = true
```
Comment (beingmattlevy) [inline deployment/gcp/sql.tf:23]: Why are we taking this off?

File: deployment/gcp/variables.tf [hunk 2]
```diff
@@ -11,7 +11,7 @@ variable "app_service_account" {
 }
 
 variable "environment" {
-  default     = "production"
+  default     = "prod"
```
Comment (beingmattlevy) [inline deployment/gcp/variables.tf:14]: Why rename to short? This is production in other areas. Is this a GCP convention?

File: deployment/gcp/sql.tf (hunk 1 above)
Comment (ev1lm0nk3y) [reply deployment/gcp/sql.tf:23]: so that I can destroy and recreate until we get the instance name right

File: deployment/gcp/variables.tf (hunk 2 above)
Comment (ev1lm0nk3y) [reply deployment/gcp/variables.tf:14]: I thought the instance name length couldn't be bigger than 32 characters, turns out that I was wrong here. I have reverted.

File: deployment/gcp/sql.tf (hunk 1 above)
Comment (ev1lm0nk3y) [reply deployment/gcp/sql.tf:23]: I will make sure this is turned on once we are confirmed with the correct naming

**Comment by ev1lm0nk3y** [comment]
I've reverted the "prod" nomenclature

**Review by beingmattlevy — APPROVED** [review]

**Pull request merged**